packages/@aws-cdk/aws-elasticloadbalancingv2/lib/shared/base-target-group.ts

```diff
@@ -365,3 +360,9 @@ export interface LoadBalancerTargetProps {
    */
   targetJson?: any;
 }
+
+export function loadBalancerNameFromListenerArn(listenerArn: string) {
+    // arn:aws:elasticloadbalancing:us-west-2:123456789012:listener/app/my-load-balancer/50dc6c495c0c9188/f2f7dc8efc522ab2
```

Comment:
Would prefer this to be part of the function JSDoc as an example, and I think it'd be nice to have the corresponding output, too (aka what part are we extracting in fact).

Reply:
Fair